Add binary.dart

```diff
@@ -0,0 +1,37 @@
+//Qa(67)
+class Solution {
+  String addBinary(String a, String b) {
+    if (a.length != b.length) {
+      if (a.length > b.length) {
+        b = List.filled(a.length - b.length, (0)).join('') + b;

```

Comment:
this is a good approach!
in dart there is a cool shorthand for that 
if you want to repeat a string you can multiply it by the number of times like this
`"0" * 4 = "0000"`

Reply:
it is always better to use language shorthand as they are optimized most of the time.